apps/web/pages/settings/my-account/profile.tsx

```diff
@@ -236,6 +240,9 @@ const ProfileView = () => {
     name: user.name || "",
     email: user.email || "",
     bio: user.bio || "",
+    linkedinLink: ((user?.metadata as Prisma.JsonObject)?.linkedinLink as string) || "",
+    XLink: ((user?.metadata as Prisma.JsonObject)?.XLink as string) || "",
+    metadata: { linkedinLink: "", XLink: "" },
```

Comment:
i think we can come up with a better schema here. 

 i don't think we should make one column for each socialLink. maybe an object? 

lets make this more generic and not specifically about X or Linkedin.

maybe 

socialLinks = {
 icon: X
 link: "https://x.com/peer_rich"
}

Reply:
@Abhinav-Developer-23 I agree with Peer. We should group LinkedIn and X under "socials":
1. Wherever we need "user.metadata" have "user.metadata.socials" object that stores links for LinkedIn and X.
2. Wherever "metadata" does not appear, just have "socials" object that stores links for LinkedIn and X.

What do you think @Abhinav-Developer-23 ?